qoffer-infrastructure/src/main/resources/log4j2.xml

```diff
@@ -1,15 +0,0 @@
-<Configuration status="INFO">
-    <Appenders>
-        <Console name="Console" target="SYSTEM_OUT">
-            <PatternLayout pattern="%d{dd.MM.yyyy HH:mm:ss} [%level] [%logger{1}] - %msg%n"/>
-        </Console>
-    </Appenders>
-    <Loggers>
-        <Logger name="life.qbic" level="info" additivity="false">
-            <AppenderRef ref="Console"/>
-        </Logger>
-        <Root level="all">
-            <AppenderRef ref="Console"/>
-        </Root>
-    </Loggers>
-</Configuration>
```

Comment:
Why remove this file?

Reply:
Did I?
I think that was a mistake. I will put it back